eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -5,7 +5,7 @@ parameters:
 resources:
   containers:
     - container: linux_arm
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-arm
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-cross-arm-0230407130525-290f679
```

Comment:
Why are we using the versioned container IDs? Didn't we move to the versionless ones to ensure we'd always consume security updates automatically?